app/controllers/DatasetController.scala

```diff
@@ -224,28 +224,42 @@ class DatasetController @Inject()(userService: UserService,
       implicit ctx: DBAccessContext,
       m: MessagesProvider): Fox[List[JsObject]] =
     for {
+      _ <- Fox.successful(())
+      _ = logger.info(s"datasets: $datasets, requestingUser: $requestingUser")
```

Comment:
```suggestion
      _ = logger.info(s"datasets: $datasets, requestingUser: ${requestingUser._id}")
```

Reply:
> let’s avoid printing entire user objects,

only user objects?

Reply:
Yes, should be ok. This is only temporary, this isn’t super sensitive data and the logs are of course not public.